src/controllers/chatController.ts

```diff
@@ -45,3 +59,38 @@ async function getChatFromDB(gameIndex: number): Promise<object[]> {
     throw err;
   }
 }
+
+/**
+ * Returns the current UTC time as a readable string.
+ * E.g. "2019-08-19 23:53:49"
+ */
+function getCurrentTime(): string {
```

Comment:
I think we have another copy of this function in the high score code. Should we unify them?

Reply:
I opted to remove this with the schema change in #22.

Reply:
(waiting to review this part until #22 is merged)